README.md

```diff
@@ -3172,6 +3172,7 @@ _Software written in Go._
 
 - [abbreviate](https://github.com/dnnrly/abbreviate) - abbreviate is a tool turning long strings in to shorter ones with configurable separators, for example to embed branch names in to deployment stack IDs.
 - [aptly](https://github.com/smira/aptly) - aptly is a Debian repository management tool.
+- [asciinema](https://github.com/gvcgo/asciinema) - A cross-platform asciinema(v2) terminal session recorder for MacOS/Linux/Windows.
```

Comment:
I won't accept this Asciinema because maintaining a separate Go fork seems unnecessary given the active development of the original project. Also, your README doesn't mention that it's an unofficial Go fork based on Asciinema's own Golang code from securisec/asciinema, which is GPL licensed. Your project being MIT licensed is considered a violation since you can't change licenses like that.